jetstream/engine/token_utils.py

```diff
@@ -40,9 +41,10 @@ def take_nearest_length(lengths: list[int], length: int) -> int:
   return lengths[pos]
 
 
-def tokenize_and_pad(
```

Comment:
Can we still keep this function?

Reply:
Any particular reason to keep the function?

I changed it to pad_tokens because different tokenizers use different function calls to tokenize the strings. It's easier to have a function just for padding and let each tokenizer convert strings to token IDs.

Reply:
I do agree separate two part will be clear, meanwhile it bring two functions instead of just one functions to tokenizer and pad. Engine implementation still use this tokenize_and_pad (i.e. interactive mode from pytorch side), it easier for client if the API is consistency from different versions. I feel it would be better to keep both of them and let client decide which one they would like to use. In general, this is a nit comments and won't block your current PR, some one can change this later.

Reply:
Thank you. I have a PR to update the pytorch interactive mode. So, that should fix it.